packages/cli/test/clone-example.test.js

```diff
@@ -3,55 +3,55 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-('use strict');
+'use strict';
 
 const promisify = require('util').promisify;
 
 const cloneExampleFromGitHub = require('../generators/example/clone-example');
 const expect = require('@loopback/testlab').expect;
+const fs = require('fs');
 const TestSandbox = require('@loopback/testlab').TestSandbox;
 const glob = promisify(require('glob'));
 const path = require('path');
 
 const VALID_EXAMPLE = 'getting-started';
-const SANDBOX_PATH = path.resolve(__dirname, 'sandbox');
+const SANDBOX_PATH = path.resolve(__dirname, '..', '.sandbox');
 let sandbox;
 
-describe.skip('cloneExampleFromGitHub (SLOW)', function() {
+describe('cloneExampleFromGitHub (SLOW)', function() {
```

Comment:
You can remove the word `function` and use `describe('cloneExampleFromGitHub (SLOW)', () => {` to be consistent with the code base.

Reply:
We cannot - the next line is calling `this.timeout()`. This is one of the gotchas of using arrow functions with Mocha.